### DISCUSSION TITLE
How to change my interests in Explore tab?

### DISCUSSION THREAD

**Discussion by Lippiece** [community/community#38561] [Other Feature Feedback, Questions, & Ideas] [answered]
Currently, it's showing me all kinds of Minecraft repos I don't care about. I also don't have any Minecraft repo starred.

**Comment by bogomil** [comment, marked as answer]
I think if you go here https://github.com/explore and star some topics you are interested in, the view will change. Could you give it a try and let us know?

**Comment by subidit** [comment]
The GitHub explore tab shows you repos based on your interest. You should try starring a few more topics or starring some repos to give it some idea about your interest. You could find all your starred repos and topics here - https://github.com/stars

Find some new topics to star here - https://github.com/topics

Good luck.

**Comment by carterlasalle** [comment]
@Lippiece The same thing is happening to me. How did you fix it>

**Reply by Lippiece** [reply]
Please refer to https://github.com/orgs/community/discussions/38561#discussioncomment-4086585.

Just star what you want to see and unstar what you don't
